VimeoNetworking/VimeoNetworking/VimeoClient.swift

```diff
@@ -388,6 +387,8 @@ final public class VimeoClient
         }
         catch let error
         {
+            self.responseCache.removeResponseForRequest(request)
```

Comment:
One possible error that may want to hold on to the cache is something like a 'No Internet' error, but I believe we're already handling that upstream of this logic in that we don't even attempt to make a request when we have no connection.  I can't really think of any other error at the moment that would fall into the same pattern.